src/utils/generation_utils.py

```diff
@@ -139,9 +139,11 @@ def generate_likelihood_estimation(
     return block
 
 
-def generate_quest_summary(quest_name: str, context: AgentContext) -> Optional[Block]:
+def generate_quest_summary(quest_name: str, context: AgentContext, failed: bool = False) -> Optional[Block]:
     """Generates and sends a quest summary to the player."""
     prompt = "Please summarize the above quest in one to two sentences."
+    if failed:
+        prompt += " Indicate that the quest was ultimately failed due to exhaustion."
```

Comment:
question: Does the "due to exhaustion" part of this prompt show itself in the failure summaries?  Especially since it's last in the prompt, I can imagine this is the kind of thing the LLM would key into heavily, so you'd end up with descriptions that frequently indicated that the main character got too tired to continue.  Not sure if that was your intent?

Reply:
It does have an influence (from the transcript posted to slack).

Maybe we could just leave it more ambiguous and just let the LLM decide how to describe it given the ChatHistory?

Reply:
It was my intent, to match the flavor of failing due to failing too often (as opposed to failing catastrophically once).  I can remove it for comparisons sake rq